packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -106,6 +107,9 @@ class Serverpod {
   /// Serverpod runtime settings as read from the database.
   internal.RuntimeSettings get runtimeSettings => _runtimeSettings!;
 
+  /// Whether to serialize enums as Strings rather than as ints
+  late bool serializeEnumValuesAsStrings;
```

Comment:
@vlidholt I figured out how to totally remove the runtime dependency upon `generator.yaml` -- see the latest commit.

The remaining issue surrounds `GeneratorConfig.instance`. This is set to the *most recent instantiation* of `GeneratorConfig`, which is ugly (but which in practice should not cause a problem, because there should not be two `GeneratorConfig`s being used at the same time, given that `serverpod_cli` only processes one generator task at a time).

Reply:
@vlidholt one more thing -- currently `lib/src/generated/log_entry.dart` fails to compile in the github tests. I can't run `tools/generate_all` on my project and then check in the results, because around 25 generated files differ significantly from the versions checked in to git (#1076). You really need to find a way to not include generated files in git.